<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkUSCombinedModalityManagerWidget</class>
 <widget class="QWidget" name="QmitkUSCombinedModalityManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ultrasound &amp;&amp; Tracking Devices</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Ultrasound Devices:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkServiceListWidget" name="m_USDevices" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Tracking Devices:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkServiceListWidget" name="m_TrackingDevices" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_CreateCombinedModalityButton">
        <property name="text">
         <string>Create Combined Modality</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Combined Modalities</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QmitkServiceListWidget" name="m_CombinedModalities" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_RemoveCombinedModalityButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkServiceListWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkServiceListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_CreateCombinedModalityButton</sender>
   <signal>clicked()</signal>
   <receiver>QmitkUSCombinedModalityManagerWidget</receiver>
   <slot>OnCreateCombinedModalityButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_RemoveCombinedModalityButton</sender>
   <signal>clicked()</signal>
   <receiver>QmitkUSCombinedModalityManagerWidget</receiver>
   <slot>OnRemoveCombinedModalityButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnCreateCombinedModalityButtonClicked()</slot>
  <slot>OnRemoveCombinedModalityButtonClicked()</slot>
 </slots>
</ui>
